packages/grafana-ui/src/components/Logs/LogRowContextProvider.tsx

```diff
@@ -160,11 +161,19 @@ export const LogRowContextProvider: React.FunctionComponent<LogRowContextProvide
         let hasMoreLogsBefore = true,
           hasMoreLogsAfter = true;
 
-        if (currentResult && currentResult.data[0].length === value.data[0].length) {
+        const currentResultBefore = currentResult?.data[0][0];
```

Comment:
Can we just fix the dimensionality [here](https://github.com/grafana/grafana/blob/f48ba11d4cc8eb413d6cb0f6b409fe3b2dc7710b/packages/grafana-ui/src/components/Logs/LogRowContextProvider.tsx#L106)? Seems like returning the logs wrapped in one additional array is just a mistake in that part of that code.